src/commands/support/simbridgeDebug.ts

```diff
@@ -11,7 +11,7 @@ const simbridgeDebugEmbed = makeEmbed({
         '',
         '1. Open the folder where SimBridge is installed.',
         '2. Open a terminal window in that location (Hold `SHIFT` and right click on the empty space of the folder, then select "Open Windows Terminal", "Open Command Prompt", or "Open Powershell Window".',
-        '3. Type `fbw-simbridge.exe` and press enter.',
+        '3. Type `.\fbw-simbridge.exe` and press enter.',
```

Comment:
```suggestion
        '3. Type `.\\fbw-simbridge.exe` and press enter.',
```

Have you tested your change? See image for the result of your change.

![image](https://github.com/flybywiresim/discord-bot/assets/67422707/5f159f37-ff91-4da0-87dc-c87becd3682f)

Reply:
Yeah, didn't have time to test yet